src/components/TurnTable/index.js

```diff
@@ -1,25 +1,32 @@
 import React from 'react';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
 import PropTypes from 'prop-types';
+import '../MyBoard/styles.css';
 
 import Turn from './Turn';
 
-function TurnTable({turnRecords}) {
+function TurnTable({ header, turnRecords }) {
+    const turns = turnRecords.length;
```

Comment:
turnLength

Reply:
This variable (turnLength) is gone for now.  Will most likely come back later when we add back functionality.